bin/vue-build

```diff
@@ -150,7 +150,9 @@ var webpackConfig = {
       cwd('node_modules'), // modules in cwd's node_modules
       ownDir('node_modules') // modules in package's node_modules
     ],
-    alias: {}
+    alias: {
+      '@': 'src'
```

Comment:
We use an absolute path for this alias in the webpack template as well:

https://github.com/vuejs-templates/webpack/blob/master/template/build/webpack.base.conf.js#L27

/ping @posva